website/print_fd_page.php

```diff
@@ -8,7 +8,16 @@ function print_fd_page($date,$type){
 //	$all_img_paths = find_latest_file($date,$inst,$filter,"png","fd",NULL,"all");
 	//var_dump($all_img_paths);
 //	$img_path= find_latest_file($date,$inst,$filter,"png","thmb",NULL,"latest","full");
+
+	if ($filter != "chimr")
+	{
 	$img_path= find_latest_file($date,$inst,$filter,"png","fd");
+	}
+	else
+	{
+	$img_path= find_latest_file($date,$inst,$filter,"png","ch");
+	}
+	
```

Comment:
I would prefer to have just the `if` to change the `fd` or `ch` bit and keep the rest the same:
```php
$img_type = ($filter != "chimr")? 'fd' : 'ch';
$img_path= find_latest_file($date,$inst,$filter,"png",$img_type);
```
So there's only one line to update if needed in the future and less to keep control of.